server/issue.go

```diff
@@ -1006,3 +1008,109 @@ func (s *Server) postInboxIssueActivity(ctx context.Context, issue *api.Issue, a
 
 	return nil
 }
+
+func (s *Server) getTenantDatabases(ctx context.Context, projectID int, databaseList []*api.Database, databaseName string) ([][]*api.Database, error) {
```

Comment:
getTenantDatabaseMatrix.

Matrix is more proper to indicate 2 dimensional array.